content/docs/sidebar.json

```diff
@@ -139,7 +139,7 @@
         "label": "Experiment Management",
         "slug": "experiment-management",
         "source": "experiment-management/index.md",
-        "children": ["checkpoints"]
+        "children": ["sharing", "checkpoints"]
```

Comment:
Q: should we make it also Sharing Experiments (I know - we have Experiments ... Experiments in the side panel, but just "sharing" feels a bit weird)?

Reply:
Right. That's better.